src/bcrypt/__init__.py

```diff
@@ -103,3 +125,14 @@ def kdf(password, salt, desired_key_bytes, rounds):
 def _bcrypt_assert(ok):
     if not ok:
         raise SystemError("bcrypt assertion failed")
+
+
+def _compare(a, b):
```

Comment:
This seems like the worst of both worlds, really. An unnecessary constant-time comparison that isn’t even necessarily used?